includes/tracks/class-wc-tracks.php

```diff
@@ -17,6 +17,18 @@ class WC_Tracks {
 	 */
 	const PREFIX = 'wca_test_';
 
+	/**
+	 * Option name for total store revenue.
+	 */
+	const REVENUE_CACHE_OPTION = 'woocommerce_tracker_orders_gross';
+
+	public static function init() {
```

Comment:
Not sure where this get initialized... Also missing docblock.

Reply:
Fixed in rebase.